torch/jit/__init__.py

```diff
@@ -1159,7 +1159,10 @@ def __init__(self, original, stubs):
                 elif isinstance(item, Parameter) or (isinstance(item, Module) and item is not self):
                     ScriptModule.__setattr__(self, name, item)
             for name in original._buffers:
-                self.register_buffer(name, original._buffers[name])
+                if original._buffers[name] is None:
```

Comment:
can you make this the same semantic with parameters, or make parameters same semantic as this

Reply:
Will do in a follow up PR just to get things landed faster